### PR TITLE
chore: move versioning outside the main package

### PR DESCRIPTION
**Pull request by vgonkivs**
## Overview
Move version flags outside the main package in order to have an access to them

```
./build/celestia version
Semantic version: v0.11.0-rc1-73-g509b74b9
Commit: 509b74b90bc6b3bf86f4730943abd3840430b03a
Build Date: Wed Jul 12 20:45:36 EEST 2023
System version: amd64/darwin
Golang version: go1.20.3
```

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: nodebuilder/node_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	collectormetricpb "go.opentelemetry.io/proto/otlp/collector/metrics/v1"
 	"google.golang.org/protobuf/proto"
 
+	"github.com/celestiaorg/celestia-node/cmd/version"
```
Comment (Wondertan) [inline nodebuilder/node_test.go:16]: Nodebuilder should not depend on cmd. I don't know where to put BuildInfo yet to fix this and open to suggestions

Comment (vgonkivs) [reply nodebuilder/node_test.go:16]: 🤔 . I think then that the best way is to revert changes related to `nodebuilder`: to have own buildInfo in `nodebuilder` package + cast from the `version.BuildInfo` to the `nodebuilder.buildInfo`

Comment (Wondertan) [reply nodebuilder/node_test.go:16]: Sounds good to me, or even avoid the BuildInfo inside of `version`

Comment (vgonkivs) [reply nodebuilder/node_test.go:16]: I thought about it, but imo it looks a bit nicer when we return struct

**Review by Wondertan — APPROVED** [review]

**Review by walldiss — APPROVED** [review]

**Review by Wondertan — APPROVED** [review]

**Review by walldiss — APPROVED** [review]

**Pull request merged**